app/components/playlist/playlist-form.tsx

```
@@ -0,0 +1,140 @@
+          disabled={isPending}
+        >
+          Cancel
+        </button>
+        <button
+          type="submit"
+          disabled={isPending || !title.trim()}
+          className="px-4 py-2 rounded bg-[--md-sys-color-primary] text-[--md-sys-color-on-primary] disabled:opacity-50"
+        >
+          {isPending ? (
+            <span className="flex items-center gap-2">
+              <span className="animate-spin"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)